src/test/rustdoc-ui/check-doc-attrs.stderr

```diff
@@ -0,0 +1,71 @@
+error: this attribute can only be applied to a `use` item
+  --> $DIR/check-doc-attrs.rs:3:8
+   |
+LL | #![doc(inline)]
+   |        ^^^^^^
+   |
```

Comment:
Unnecessary nitpick: The only inconsistency I see here is that we don't point at the `mod`, which is reasonable, but maybe we would want to add a `note` saying "`#[doc(inline)]` cannot be applied to a `mod`, or something similar (but we'd probably want another iteration to make sure we remove redundant wording... maybe add it to the message?: "this attribute can only be applied to a `use` item, not a `mod`").